posts/specfile-advanced/index.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Advanced usage of the specfile library
+date: 2025-01-09
+authors: nforro
+tags:
+  - specfile
+---
+
+# Advanced usage of the specfile library
+
+Are you already familiar with the basic concepts of the _specfile_ library?
+Then welcome to a collection of some slightly more advanced topics that you could find useful.
+Otherwise you can take a look at [Introduction to specfile library](posts/specfile-introduction).
+
+<!--truncate-->
+
+## Forced parsing
+
+Imagine you want to make changes to a spec file that requires some sources
```

Comment:
#off-topic is there some official glossary with the spelling? :smile: I've seen

* spec file
* specfile
* spec-file

Still not sure which one is _the right one_. :D